src/templates/layouts/event/event.stories.tsx

```diff
@@ -0,0 +1,926 @@
+import React from 'react'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import { Event } from '.'
+
+export default {
+  title: 'Layouts/Event',
+  component: Event,
+} as ComponentMeta<typeof Event>
+
+const Template: ComponentStory<typeof Event> = (args) => <Event {...args} />
+
+export const EventExample = Template.bind({})
+EventExample.args = {
```

Comment:
this feels like it's probably too many args, very likely doesn't need all of this data in order to render (like the nested breadcrumbs on facilityLocation, etc). not a blocker, we can clean it up later... i think most stories need some cleanup